packages/core/docs/advanced/ssr-cache.md

```diff
@@ -0,0 +1,144 @@
+# Server Side Rendering Cache
+
+## Introduction
+
+Caching allows saving pages rendered on the server for later use, to avoid computationally expensive rendering from scratch when possible. This is especially useful when the application has pages that require a lot of computation, make many API calls, or change infrequently. It not only reduces the load on the server but also greatly improves performance.
+
+Caching SSR output in Vue Storefront requires two packages:
+* `@vue-storefront/cache` - Nuxt.js module, that does the heavy lifting. It registers required plugins, creates [invalidation endpoint](#invalidating-cache), and hooks into the render cycle. 
+* **the driver** - thin layer on top of `@vue-storefront/cache` that integrates with specific caching solution, such as [Redis](https://redis.io/) or [Memcached](https://memcached.org/).
+
+Vue Storefront team provides [integration with Redis](../integrations/redis-cache.md).
+
+If you'd like to know how to build your own driver, [see this page](../integrate/cache-driver.md).
+
+## Installation
+
+### Add dependencies
+
+To enable caching in Vue Storefront you will need two packages - `@vue-storefront/cache` and driver package.
+
+```sh
+yarn add @vue-storefront/cache
+yarn add <DRIVER-NAME> # eg. @vsf-enterprise/redis-cache
+```
+
+### Configure Nuxt
+
+The next step is to register `@vue-storefront/cache` package as a module in `nuxt.config.js` with driver and invalidation configuration.
```

Comment:
But we are registering `@vue-storefront/cache/nuxt`. It might be confusing for users